docs/byoh-instance-pre-requisites.md

```diff
@@ -4,11 +4,10 @@ The following pre-requisites must be fulfilled in order to add a Windows BYOH no
 * The Docker container runtime must be installed on the instance.
 * The instance must be on the same network as the Linux worker nodes in the cluster.
 * Port 22 must be open and running [an SSH server](https://docs.microsoft.com/en-us/windows-server/administration/openssh/openssh_install_firstuse).
-  * The default shell for the SSH server must be set to [cmd.exe](https://docs.microsoft.com/en-us/windows-server/administration/openssh/openssh_server_configuration#configuring-the-default-shell-for-openssh-in-windows).
 * Port 10250 must be open in order for log collection to function.
 * An administrator user is present with the [private key used in the secret](/README.md#create-a-private-key-secret) set as an authorized SSH key.
 * The hostname of the instance must follow the [RFC 1123](https://datatracker.ietf.org/doc/html/rfc1123) DNS label standard:
   * Contain only lowercase alphanumeric characters or '-'.
   * Start with an alphanumeric character.
   * End with an alphanumeric character.
-* A PTR record must exist corresponding to the instance address which resolves to the instance hostname for successful reverse DNS lookups. 
\ No newline at end of file
+* A PTR record must exist corresponding to the instance address which resolves to the instance hostname for successful reverse DNS lookups.
```

Comment:
looks like just a extra space got remove . can we undo this ?

Reply:
LGTM; IMO fixing a surrounding typo does not hurt.

Reply:
While I am fine with this instance, fixing surrounding typos needs to be taken on a case by case basis. I am seeing other examples of more invasive changes and excuse is that I was in the area.